plugins/core/gov.nasa.jpl.imce.oml.zip/src/gov/nasa/jpl/imce/oml/zip/OMLSpecificationTables.xtend

```diff
@@ -2758,6 +2778,66 @@ class OMLSpecificationTables {
     return bos.toByteArray()
   }
   
+  static def byte[] subDataPropertyOfAxiomsByteArray(Extent e) {
+  	val ByteArrayOutputStream bos = new ByteArrayOutputStream()
+  	val PrintWriter pw = new PrintWriter(bos)
+  	OMLTables.subDataPropertyOfAxioms(e).forEach[it |
+  	  pw.print("{")
+      pw.print("\"uuid\":")
+      pw.print("\"")
+      pw.print(it.uuid())
+      pw.print("\"")
+      pw.print(",")
+      pw.print("\"tboxUUID\":")
+      pw.print("\"")
+      pw.print(it.tbox.uuid())
+      pw.print("\"")
+      pw.print(",")
+      pw.print("\"subPropertyUUID\":")
+      pw.print("\"")
+      pw.print(it.subProperty.uuid())
+      pw.print("\"")
+      pw.print(",")
+      pw.print("\"superPropertyUUID\":")
+      pw.print("\"")
+      pw.print(it.superProperty.uuid())
+      pw.print("\"")
+      pw.println("}")
+    ]
+    pw.close()
+    return bos.toByteArray()
+  }
+  
+  static def byte[] subObjectPropertyOfAxiomsByteArray(Extent e) {
+  	val ByteArrayOutputStream bos = new ByteArrayOutputStream()
+  	val PrintWriter pw = new PrintWriter(bos)
+  	OMLTables.subObjectPropertyOfAxioms(e).forEach[it |
+  	  pw.print("{")
+      pw.print("\"uuid\":")
+      pw.print("\"")
+      pw.print(it.uuid())
+      pw.print("\"")
+      pw.print(",")
+      pw.print("\"tboxUUID\":")
+      pw.print("\"")
+      pw.print(it.tbox.uuid())
+      pw.print("\"")
```

Comment:
Line 2821-2824 all other similar function do not have this reference. I am not sure if it is required. Please kindly check.

Reply:
If the above mentioned lines are removed go to line 5333 to make reflect the changes.

Reply:
Ditto